README.md

```diff
@@ -583,6 +583,66 @@ _You can enable the following settings in Xcode by running [this script](resourc
     listingUrgencyBookedShortRowContent(),
   ]
   ```
+  
+* <a id='multi-line-conditions'></a>(<a href='#multi-line-conditions'>link</a>) **Multi-line conditional statements should break after the leading keyword.** Indent each individual statement by [2 spaces](https://github.com/airbnb/swift#spaces-over-tabs). Put the open curly brace on the next line. In `guard` statements, place the `else` keyword on the same line as the open curly brace. [![SwiftFormat: wrapArguments](https://img.shields.io/badge/SwiftFormat-wrapArguments-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#wrapArguments) [![SwiftFormat: wrapMultilineStatementBraces](https://img.shields.io/badge/SwiftFormat-wrapMultilineStatementBraces-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#wrapMultilineStatementBraces) [![SwiftFormat: elseOnSameLine](https://img.shields.io/badge/SwiftFormat-elseOnSameLine-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#elseonsameline)
```

Comment:
Fair!

Reply:
One concern I have, though, is that it's not very clear how _"Put the open curly brace on the next line."_ applies to `guard` statements without guidance on where to place the `else` keyword.

Reply:
I suppose we could have three completely separate rules here:
 1. Multi-line conditional statements should break after the leading keyword.
 2. In multiline statements, put the open curly brace on its own line.
 3. In `guard` statements, place the `else` keyword on the same line as the open curly brace.

Reply:
@calda after reading this PR more closely, maybe I was mistake in https://github.com/airbnb/swift/pull/108/files#r512120282. Since we are only enabling the `--guardelse` option of the `elseOnSameLine` rule, maybe there isn't as much scope creep as I thought...

Reply:
Re: https://github.com/airbnb/swift/pull/108/files#r512139531 I wonder if 1) can be a rule and then 2), 3), and 4) can be a more generic rule about multiline control flow statements.

Reply:
`elseOnSameLine`'s `--elseposition` option doesn't have a `preserve` case like some other options, so it isn't possible to use `--guardposition` without also using `--elseposition` at the moment.

You're right that we probably shouldn't adopt the `elseOnSameLine` rule without specifying how to handle the other cases it applies to.

Reply:
If we took [this](https://github.com/airbnb/swift/pull/108/files#r512314810) approach it seems like we'd want to adopt both the `--guardelse` and `--elseposition` options of the `elseOnSameLine` rule, if I understand how it works correctly.

Reply:
I'll update this PR to be specifically focused on (1), and we can tackle (2), (3), and (4) separately.

Reply:
> `elseOnSameLine`'s `--elseposition` option doesn't have a `preserve` case like some other options, so it isn't possible to use `--guardposition` without also using `--elseposition` at the moment.

@calda I don't see any mention of `--elseposition` in `airbnb.swiftformat` at the moment. Is it implicitly enabled, or is that an oversight?

Reply:
`--elseposition` defaults to `same-line`

From the [SwiftFormat docs](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#elseonsameline):
> `--elseposition` `Placement of else/catch: "same-line" (default) or "next-line"`

Reply:
@bachand I updated this PR to only include _"Multi-line conditional statements should break after the leading keyword. Indent each individual statement by 2 spaces."_

Reply:
> `--elseposition` defaults to `same-line`

Oh I assumed that `--elseposition` only had a default if you actually specified the option.